lib/ansible/modules/system/aix_inittab.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Joris Weijters <joris.weijters@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+
+DOCUMENTATION = '''
+---
+author: "Joris Weijters (@molekuul)"
+module: aix_inittab
+short_description: Manages the inittab at AIX.
+description:
+    - Manages the inittab at AIX.
+version_added: "2.3"
+options:
+  name:
+    description: Name of the inittab entry.
+    required: True
+    alias: service
+    type: string
+    default: null
+  runlevel:
+    description: Runlevel of the entry.
+    required: True
+    type: string
+    default: null
+  processaction:
+    description: Action what the init has to do with this entry.
+    required: True
+    choices: [
+               'respawn',
+               'wait',
+               'once',
+               'boot',
+               'bootwait',
+               'powerfail',
+               'powerwait',
+               'off',
+               'hold',
+               'ondemand',
+               'initdefault',
+               'sysinit'
+              ]
+    type: string
+    default: null
+  command:
+    description: What command has to run.
+    requred: True
+    type: string
+    default: null
+  follow:
```

Comment:
The inittab at AIX is run top down, it looks like:
[ Identifier ] : [ RunLevel ] : [ Action ] : [ Command ]
Depending on the action, the init proces will run a command once and not wait for it to finnish, or wil run a command once and wait for it. possible actions are:
* respawn:   If the process identified in this record does not exist, start the process. If the process currently exists, do nothing and continue scanning the /etc/inittab file.
* wait:  When the init command enters the run level specified for this record, start the process and wait for it to stop. While the init command is in the same run level, all subsequent reads of the /etc/inittab file ignore this object.
* once: When the init command enters the run level specified for this record, start the process, do not wait for it to stop and when it does stop do not restart the process. If the system enters a new run level while the process is running, the process is not restarted.
* boot: Read this record only when the system boots and reads the /etc/inittab file. The init command starts the process. Do not wait for the process to stop and when it does stop, do not restart the process. The run level for this process should be the default, or it must match the run level specified by the init command at startup time.
* bootwait: Read this record only when the system boots and reads the /etc/inittab file. The init command starts the process. Wait for it to stop, and when it does stop, do not restart the process.
* powerfail: Start the process identified in this record only when the init command receives a SIGPWR power fail signal.
* powerwait: Start the process identified in this record only when the init command receives a SIGPWR power fail signal, and wait until it stops before continuing to process the /etc/inittab file.
* off: If the process identified in this record is currently running, send the warning signal SIGTERM and wait 20 seconds before sending the SIGKILL kill signal. If the process is nonexistent, ignore this line.
* hold: When the process identified in this record is terminated, do not start a new one. The hold action can only be activated by the phold command.
* ondemand: Functionally identical to respawn. If the process identified in this record does not exist, start the process. If the process currently exists, do nothing and continue scanning the /etc/inittab file. Specify this action to perform the respawn action when using a, b, or c run levels.
* initdefault: A line with this action is processed only when the init command is originally invoked. The init command uses this line to determine which run level to originally enter. The command does this by taking the highest run level specified in the RunLevel parameter and using that as the command's initial state. If the RunLevel parameter is empty, its value is interpreted as 0123456789, and the init command enters a run level of 9. If the init command does not find an initdefault line in the inittab file, it requests an initial run level from the operator at initial program load (IPL) time.
* sysinit: Start the process identified in this record before the init command tries to access the console. For example, you might use this to initialize devices.